src/haddock/modules/refinement/emref/cns/symmultimer.cns

```diff
@@ -21,11 +21,11 @@ if ($Data.flags.sym eq true) then
     while ($ncount < $data.numc2sym) loop c2symloop
 
         eval ($ncount = $ncount + 1)
-        evaluate ($i1start = decode($Toppar.c2sym_sta1_$ncount))
-        evaluate ($i1end   = decode($Toppar.c2sym_end1_$ncount))
+        evaluate ($i1start = $Toppar.c2sym_sta1_$ncount)
+        evaluate ($i1end   = $Toppar.c2sym_end1_$ncount)
         evaluate ($chain1  = $Toppar.c2sym_seg1_$ncount)
-        evaluate ($i2start = decode($Toppar.c2sym_sta2_$ncount))
-        evaluate ($i2end   = decode($Toppar.c2sym_end2_$ncount))
```

Comment:
what was the purpose of this `decode`?

Reply:
Before those variables were define with double quotes making them strings. CNS has a decode/encode function to convert strings to numbers and vice-versa